### ISSUE TITLE
XNNPACK delegate not enabled in TensorFlow Lite Python Interpreter.

### ISSUE THREAD

**Issue by NobuoTsukamoto** [tensorflow/tensorflow#53042]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow): No.
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Ubuntu 18.04 (x64) and Raspberry Pi OS 64bit (raspios_arm64-2021-04-09)
- Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device: Raspberry Pi 4
- TensorFlow installed from (source or binary): Source
- TensorFlow version (use command below): v2.7.0 and 7b290f9fd9fbf2ac4352b3cbe327e1067e5a3574
- Python version: 3.7.3 (Raspberry Pi OS 64bit)
- Bazel version (if compiling from source): - (Build with CMake)
- GCC/Compiler version (if compiling from source): 8.3.0 (Raspberry Pi OS 64bit)
- CUDA/cuDNN version: - 
- GPU model and memory: -

**Describe the current behavior**
XNNPACK delegate not enabled in TensorFlow Lite Python Interpreter. 
Building with either CMake or Bazel will not take effect.
The following log is not output.
> INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
  
  
Delegate lazy initialization was included in the 3d3c6db1ca2d50f6f07722cd800144f8f736167c commit.
For C ++ IF, Interpreter::AllocateTensors calls ApplyLazyDelegateProviders to enable the XNNPACK delegate.
https://github.com/tensorflow/tensorflow/blob/v2.7.0/tensorflow/lite/interpreter.cc#L176

However, for Python IF, the XNNPACK delegate is not enabled because ApplyLazyDelegateProviders is not called in InterpreterWrapper::AllocateTensors.
https://github.com/tensorflow/tensorflow/blob/v2.7.0/tensorflow/lite/python/interpreter_wrapper/interpreter_wrapper.cc#L259

**Describe the expected behavior**
The XNNPACK delegate is enabled in the TensorFlow Lite Python Interpreter.

**[Contributing](https://www.tensorflow.org/community/contribute)**

- Do you want to contribute a PR? (yes/no): Yes
- Briefly describe your candidate solution(if contributing): 

**Standalone code to reproduce the issue**

**Other info / logs** Include any logs or source code that would be helpful to

**Comment by tilakrayal** [comment]
@NobuoTsukamoto ,
The issue will move to closed status once the PR is merged.

**Comment by multiverse-tf** [comment]
Many thanks for catching this issue and contributing the fix! Just posted a comment to https://github.com/tensorflow/tensorflow/pull/53044/

**Comment by NobuoTsukamoto** [comment]
I was able to delegate XNNPACK with 365a3b68471f158defc3aea79a25fdaa56be4ac8 commits.
- I built it with build_pip_package_with_cmake.sh on Raspberry Pi OS 64 bit.
- I confirmed that the following log is output.
  INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
- The value specified for the num_threads argument of tf.lite.Interpreter was specified in the thread pool of XNNPACK.


Thank you very much. I would like to close this issue.
